### ISSUE TITLE
Regression with `abook` address completion

### ISSUE THREAD

**Issue by BehindTheBrain** [neomutt/neomutt#4486]
Address autocompletion with `abook` used to quote names, as of yesterday's updates (Fedora current), `neomutt` removes the quotes, breaking abook integration. Standard `mutt` still works the same, quoting auto-completed addresses from abook, indicating to me that the change was not with `abook`, but with `neomutt`.

## NeoMutt Version

```
╰─>$ neomutt -v
NeoMutt 20241212
Copyright (C) 2015-2024 Richard Russon and friends
NeoMutt comes with ABSOLUTELY NO WARRANTY; for details type 'neomutt -vv'.
NeoMutt is free software, and you are welcome to redistribute it
under certain conditions; type 'neomutt -vv' for details.

System: Linux 6.6.63-1.qubes.fc37.x86_64 (x86_64)
ncurses: ncurses 6.4.20240127 (compiled with 6.4.20240127)
libidn2: 2.3.7 (compiled with 2.3.7)
GPGME: 1.23.2
GnuTLS: 3.8.6
libnotmuch: 5.6.0
PCRE2: 10.44 2024-06-07
storage: tokyocabinet, lmdb
compression: lz4, zlib, zstd

Configure options: --build=x86_64-redhat-linux-gnu --host=x86_64-redhat-linux-gnu --program-prefix= --disable-dependency-tracking --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --datadir=/usr/share --includedir=/usr/include --libdir=/usr/lib64 --libexecdir=/usr/libexec --localstatedir=/var --sharedstatedir=/var/lib --mandir=/usr/share/man --infodir=/usr/share/info CC=gcc SENDMAIL=/usr/sbin/sendmail ISPELL=/usr/bin/hunspell --autocrypt --disable-idn --full-doc --gnutls --gpgme --gss --idn2 --lmdb --lua --lz4 --notmuch --pcre2 --sasl --tokyocabinet --zlib --zstd

Compilation CFLAGS: -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64 -march=x86-64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer  -std=c11 -fno-delete-null-pointer-checks -D_ALL_SOURCE=1 -D_GNU_SOURCE=1 -D__EXTENSIONS__ -D_XOPEN_SOURCE_EXTENDED -I/usr/include/libassuan2 -I/usr/include -I/usr/include -I/usr/include -DNCURSES_WIDECHAR -I/usr/include -I/usr/include/p11-kit-1 -DWITH_GZFILEOP -I/usr/include -O2

Compile options:
  +autocrypt +fcntl -flock -fmemopen +futimens +getaddrinfo +gnutls +gpgme
  -gsasl +gss +hcache -homespool +idn +inotify -locales_hack +lua +nls +notmuch
  -openssl +pcre2 +pgp +sasl +smime +sqlite +truecolor

MAILPATH="/var/mail"
PKGDATADIR="/usr/share/neomutt"
SENDMAIL="/usr/sbin/sendmail"
SYSCONFDIR="/etc"

To learn more about NeoMutt, visit: https://neomutt.org
If you find a bug in NeoMutt, please raise an issue at:
    https://github.com/neomutt/neomutt/issues
or send an email to: <neomutt-devel@neomutt.org>
```

`.muttrc`:

```
...
# Abook 
# Press 'a' when over message or in message to add to addressbook
# In compose, type name, and tab, or enter to see options for completion
set query_command= "abook --mutt-query '%s'"
macro index,pager a "<pipe-message>abook --add-email-quiet<return>" "Add this sender to Abook"
bind editor <Tab> complete-query
...
```
## Expected Behaviour

To: Name -> press tab
Used to auto-complete while preserving quotes around name:
"lastname, firstname" \<email.com\>.

## Actual Behaviour

To: Name -> press tab
Now, auto-complete canabalizes quotes around name:
lastname, firstname \<email.com\>.

## How often does this happen?

- Always

## When did it start to happen?

- When I upgraded (yesterday, to the version listed above, in Fedora repos)

**Comment by flatcap** [comment]
Sorry, but good news.
This was already reported, #4482 and fixed, #4483.

If you're happy building NeoMutt, then `[main]` should be safe to use.
If not, I'll make a new release when I sober up after New Year :-)

**Comment by BehindTheBrain** [comment]
Oof, sorry about that!
I should have done a better search on older issues,
instead of just looking through open ones :)

Have a great new year!
